### PR TITLE
Move 4.10 to use stable channel

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ VERSION := $(shell git describe --tags --dirty --always)
 # To re-generate a bundle for other specific channels without changing the standard setup, you can:
 # - use the CHANNELS as arg of the bundle target (e.g make bundle CHANNELS=preview,fast,stable)
 # - use environment variables to overwrite this value (e.g export CHANNELS="preview,fast,stable")
-CHANNELS="4.9"
+CHANNELS="stable"
```
Comment (jmencak) [inline Makefile:13]: Why was 4.9 used here before contrary to the comment above?  Or turning it on its head, why don't we use 4.10 now?  Similarly below for `DEFAULT_CHANNEL`.

**Comment by ArangoGutierrez** [comment]
@celebdor @tsorya we need top enable this PR https://github.com/openshift/release/pull/22480 
cc @dagrayvid @kpouget from the team that can ack and lgtm

**Pull request merged**